pycqed/measurement/pulse_sequences/single_qubit_tek_seq_elts.py

```diff
@@ -51,6 +51,10 @@ def one_qubit_reset(qb_name, operation_dict, prep_params=dict(), upload=True,
         swept_pulses.append(segment_pulses)
 
     seq = pulse_list_list_seq(swept_pulses, seq_name, upload=False)
+
+    # reuse sequencer memory by repeating readout pattern
+    seq.repeat_ro(f"RO {qb_name}", operation_dict)
```

Comment:
Nice and compact code here. Does it work?

Reply:
thats a wrapper of sequence.repeat() assuming all readouts are the same and thus we can use the readout pattern (n_acq_elements, 1). We should discuss further with christian but I think this should be ok for all single qubits experiments. Might be more difficult for multiqubit experiments, there I am not sure how it should be done because I don't have a perfect understanding of how the repeat pattern works.